packages/test/test-end-to-end-tests/src/test/mockDetachedBlobStorage.ts

```diff
@@ -32,11 +35,31 @@ export class MockDetachedBlobStorage implements IDetachedBlobStorage {
 	}
 }
 
+export function driverSupportsBlobs(driver: ITestDriver): boolean {
+	const driversThatSupportBlobs: string[] = ["local", "odsp"];
```

Comment:
this should be outside the function, as there is no need to redefine the array on every call.